src/mixs/schema/mixs.yaml

```diff
@@ -3813,12 +3813,12 @@ slots:
     required: true
   abs_air_humidity:
     annotations:
-      Preferred_unit: gram per gram, kilogram per kilogram, kilogram, pound
-    description: Actual mass of water vapor - mh20 - present in the air water vapor
-      mixture
+      Preferred_unit: gram per gram, kilogram per kilogram, kilogram, pound, gram per cubic meter, kilogram per cubic meter, percent
+    description: Actual mass of water vapor present in the air water vapor mixture. Can be calculated via mass of water vapor divided by the volume of the air and water vapor mixture.
     title: absolute air humidity
     examples:
       - value: 9 gram per gram
```

Comment:
I agree with those who question the validity of "9 gram per gram" as an `abs_air_humidity` measurement. In fact, I think we should just remove it/replace it with "7 gram per cubic meter" (which is roughly 40.5%. RH at 20°C)